app/views/redmine_required_field/_settings.html.erb

```diff
@@ -83,15 +91,19 @@
   var rrf_issue_statuses = <%= raw rrf_all_issue_statuses.map(&:reverse).to_h.to_json %>;
   var rrf_trackers = <%= raw rrf_all_trackers.map(&:reverse).to_h.to_json %>;
   var rrf_projects = <%= raw rrf_all_projects.map(&:reverse).to_h.to_json %>;
+  var rrf_roles = <%= raw rrf_all_roles.map(&:reverse).to_h.to_json %>;
 
   function rrf_redraw_issue_required() {
     var table_body = '';
-
     $.each(rrf_current_issue_required, function(ix, data) {
+      console.log(data);
       table_body += '<tr>' +
         '<td>' + $.map(data.project_ids, function(k, ix) {return rrf_projects[k]}).join('<br>') + '</td>' +
         '<td>' + $.map(data.tracker_ids, function(k, ix) {return rrf_trackers[k]}).join('<br>') + '</td>' +
         '<td>' + $.map(data.issue_status_ids, function(k, ix) {return rrf_issue_statuses[k]}).join('<br>') + '</td>' +
+        '<td>' + $.map(data.role_ids, function(k, ix) {return rrf_roles[k]}).join('<br>') + '</td>' +
+        '<td>' + (data.role_excluded=="1" ? '<%=l('redmine_required_field.settings.exclude_roles')%>':'') + '<br>' +
```

Comment:
please leave space after '<%=' and before '%>'

Reply:
updated on next commit